src/menuPage.js

```diff
@@ -0,0 +1,41 @@
+import htmlTag from './htmlTags';
+import imgTag from './imageTag';
+import pasta from './assets/images/menu-pasta.jpg';
+import drink from './assets/images/menu-drink.jpg';
+import fruits from './assets/images/menu-fruits.jpg';
+import salad from './assets/images/menu-salad.jpg';
+import steak from './assets/images/menu-steak.jpg';
+import sweet from './assets/images/4.jpg';
+
+export default function menuPage() {
+  const main = document.getElementById('main');
+  const menuSection = htmlTag('section', 'menu', 'page');
+  const grid = htmlTag('div', 'menu-grid', 'page');
+  const pastaImage = imgTag(pasta, 'pasta-image', 'box-image');
+  const saladImage = imgTag(salad, 'salad-image', 'box-image');
+  const drinkImage = imgTag(drink, 'drink-image', 'box-image');
+  const fruitsImage = imgTag(fruits, 'fruits-image', 'box-image');
+  const steakImage = imgTag(steak, 'steak-image', 'box-image');
+  const sweetImage = imgTag(sweet, 'sweet-image', 'box-image');
+  const pastaBox = htmlTag('div', 'pastaBox', 'grid-box', 'delicious pasta');
+  const saladBox = htmlTag('div', 'saladBox', 'grid-box', 'crunchy veg');
+  const drinkBox = htmlTag('div', 'drinkBox', 'grid-box', 'great cocktails');
+  const fruitBox = htmlTag('div', 'fruitBox', 'grid-box', 'fresh fruits');
+  const steakBox = htmlTag('div', 'steakBox', 'grid-box', 'yummy steaks');
+  const sweetBox = htmlTag('div', 'sweetBox', 'grid-box', 'sweetest sweets');
+
+  main.appendChild(menuSection);
```

Comment:
- **[OPTIONAL]** To avoid reflows every time you append a child, it's better to use [createDocumentFragment](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment).

The same can be applied to the other tabs.